packages/mdc-theme/README.md

```diff
@@ -64,205 +64,96 @@ $mdc-theme-background: #fff;
+| `--mdc-theme-text-primary-on-primary` | Primary text on top of a theme primary color background. |
+| `--mdc-theme-text-secondary-on-primary` | Secondary text on top of a theme primary color background. |
+| `--mdc-theme-text-hint-on-primary` | Hint text on top of a theme primary color background. |
+| `--mdc-theme-text-disabled-on-primary` | Disabled text on top of a theme primary color background. |
+| `--mdc-theme-text-icon-on-primary` | Icons on top of a theme primary color background. |
+| `--mdc-theme-text-primary-on-accent` | Primary text on top of a theme accent color background. |
+| `--mdc-theme-text-secondary-on-accent` | Secondary text on top of a theme accent color background. |
+| `--mdc-theme-text-hint-on-accent` | Hint text on top of a theme accent color background. |
+| `--mdc-theme-text-disabled-on-accent` | Disabled text on top of a theme accent color background. |
+| `--mdc-theme-text-icon-on-accent` | Icons on top of a theme accent color background. |
+| `--mdc-theme-text-primary-on-background` | Primary text on top of the theme background. |
+| `--mdc-theme-text-secondary-on-background` | Secondary text on top of the theme background. |
+| `--mdc-theme-text-hint-on-background` | Hint text on top of the theme background. |
+| `--mdc-theme-text-disabled-on-background` | Disabled text on top of the theme background. |
+| `--mdc-theme-text-icon-on-background` | Icons on top of the theme background. |
+| `--mdc-theme-text-primary-on-light` | Primary text on top of a light-colored background. |
+| `--mdc-theme-text-secondary-on-light` | Secondary text on top of a light-colored background. |
+| `--mdc-theme-text-hint-on-light` | Hint text on top of a light-colored background. |
+| `--mdc-theme-text-disabled-on-light` | Disabled text on top of a light-colored background. |
+| `--mdc-theme-text-icon-on-light` | Icons on top of a light-colored background. |
+| `--mdc-theme-text-primary-on-dark` | Primary text on top of a dark-colored background. |
+| `--mdc-theme-text-secondary-on-dark` | Secondary text on top of a dark-colored background. |
+| `--mdc-theme-text-hint-on-dark` | Hint text on top of a dark-colored background. |
+| `--mdc-theme-text-disabled-on-dark` | Disabled text on top of a dark-colored background. |
+| `--mdc-theme-text-icon-on-dark` | Icons on top of a dark-colored background. |
+
+### Sass Mixins, Variables, and Functions
+
+| Mixin                                           | Description |
+| ----------------------------------------------- | - |
+| `mdc-theme-prop($property, $style, $important)` | Applies a theme color to a property |
+| `mdc-theme-dark($root-selector, $compound)` | Creates a rule that is applied when the root element is within an Dark Theme context. If using the mixin on anything other than the base selector, you need to specify the base selector as a parameter. If using the mixin with a modifier class, pass `true` for the second argument. |
+
+#### Properties
```

Comment:
What are these properties and how can I use them? I don't think this clearly explained here.